src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -788,6 +817,20 @@ function build_eal_list() {
 	cert_build_list('cert', 'IPsec')
 ))->setHelp('Select a certificate previously configured in the Certificate Manager.');
 
+$section->addInput(new Form_Select(
+	'pkcs11certref',
+	'*Token Certificate',
+	$pconfig['pkcs11certref'],
+	build_pkcs11cert_list()
+))->setHelp('Select a certificate previously created on Token.');
+
+$section->addInput(new Form_Input(
+	'pkcs11pin',
+	'*Token PIN',
```

Comment:
This should also say `PKCS#11 Token` either in the option name or description.

Reply:
done